### ISSUE TITLE
Separate solr port for tests

### ISSUE THREAD

**Issue by mccalluc** [WGBH/openvault3#200]
It'd be nice be able to run tests, and then come back and still have an earlier big ingest available.

**Comment by afred** [comment]
@mccalluc Yes, I was about to make this same ticket. Why not use a `development` and `test` core. I think Blacklight w/ blacklight-jetty is set up to make that easy.
